Source/Core/Core/AchievementManager.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#ifdef USE_RETRO_ACHIEVEMENTS
+#include "Core/HW/Memmap.h"
```

Comment:
All of these headers can be moved into the cpp file, since they aren't required in the header.

The only header that needs to be present is `<string>`